pennylane/ops/qutrit/non_parametric_ops.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the ternary discrete-variable quantum operations
+that do not depend on any parameters.
+"""
+# pylint:disable=arguments-differ
+import numpy as np
+
+import pennylane as qml  # pylint: disable=unused-import
+from pennylane.operation import Operation
+
+OMEGA = np.exp(2 * np.pi * 1j / 3)
+ZETA = OMEGA ** (1 / 3)  # ZETA will be used as a phase for later non-parametric operations
+
+# Note: When Operation.matrix() is used for qutrit operations, `wire_order` must be `None` as specifying
+# an order that is expected to return a permuted and expanded matrix different from the canonical matrix
+# will lead to errors as `expand_matrix()` in `pennylane/operation.py`, which is used to compute the
+# permuted and expanded matrix from the canonical matrix, is hard coded to work correctly for qubits.
+
+
+class TShift(Operation):
+    r"""TShift(wires)
+    The qutrit shift operator
+
+    .. math:: TShift = \begin{bmatrix}
```

Comment:
Leaving a note so that this is on our radar: it just occurred to me that we'll have to start checking the rendered documentation to make sure that it looks correct. For example, you can see the rendered version of `TShift` [here](https://pennylane--2841.org.readthedocs.build/en/2841/code/api/pennylane.TShift.html). But right now there is nowhere in the docs that actually links to it; prior to the release, we'll have to go through and add them.